### ISSUE TITLE
[Bug] DOTA Evaluation Results Feedback of Task1 is obviously lower than the official result

### ISSUE THREAD

**Issue by jimuIee** [open-mmlab/mmrotate#1013]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmrotate/issues) and [Discussions](https://github.com/open-mmlab/mmrotate/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmrotate.readthedocs.io/en/1.x/notes/4_faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmrotate) or [latest version (1.x)](https://github.com/open-mmlab/mmrotate/tree/dev-1.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmrotate

### Environment

sys.platform: linux
Python: 3.7.0 (default, Oct  9 2018, 10:31:47) [GCC 7.3.0]
CUDA available: True
GPU 0,1,2,3,4,5,6,7: NVIDIA GeForce RTX 3090
CUDA_HOME: /usr/local/cuda-11.3
NVCC: Cuda compilation tools, release 11.3, V11.3.109
GCC: gcc (Ubuntu 5.5.0-12ubuntu1) 5.5.0 20171010
PyTorch: 1.11.0+cu113
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.5.2 (Git Hash a9302535553c73243c632ad3c4c80beec3d19a1e)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.3
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86
  - CuDNN 8.2
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.3, CUDNN_VERSION=8.2.0, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -DEDGE_PROFILER_USE_KINETO -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.11.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=OFF, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

TorchVision: 0.12.0+cu113
OpenCV: 4.9.0
MMCV: 1.5.3
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: 11.3
MMRotate: 0.3.4+9ea1aee

### Reproduces the problem - code sample

python tools/train.py configs/rotated_fcos/rotated_fcos_r50_fpn_1x_dota_le90.py

### Reproduces the problem - command or script

python tools/test.py configs/rotated_fcos/rotated_fcos_r50_fpn_1x_dota_le90.py work_dirs/rotated_fcos_r50_fpn_1x_dota_le90/latest.pth --format-only --eval-options submission_dir=task1

### Reproduces the problem - error message

The detection accuracy of FCOS on the DOTA dataset is much lower than the model provided by model zoo

### Additional information

This is your evaluation result for task 1 (VOC metrics):

mAP: 0.42395437677254383
ap of each class: plane:0.5423819423982649, baseball-diamond:0.5285280870317047, bridge:0.20667867900155412, ground-track-field:0.4569022199523975, small-vehicle:0.22461535065045377, large-vehicle:0.08442215012610903, ship:0.17547907678219063, tennis-court:0.6117272119996067, basketball-court:0.6934436730389884, storage-tank:0.8345323594333807, soccer-ball-field:0.36092004275259465, roundabout:0.6534027091402538, harbor:0.247273973001902, swimming-pool:0.49025477053589234, helicopter:0.24875340574286495

i have checked the config and confirmed the file is same as the official config, and the method of split is same,
i attached my log file 
[20240321_214044.log](https://github.com/open-mmlab/mmrotate/files/14717688/20240321_214044.log)

**Comment by Aocide119** [comment]
Maybe you didn't use the augumented data. By the way we encountered an issue with the DOTA V1.0 evaluation server, we can't reach the below links.
![image](https://github.com/open-mmlab/mmrotate/assets/101000621/7bfd160b-a95d-4f78-8c95-2244e66f64c3)

**Comment by jimuIee** [comment]
> Maybe you didn't use the augumented data. By the way we encountered an issue with the DOTA V1.0 evaluation server, we can't link to the below links. ![image](https://private-user-images.githubusercontent.com/101000621/316037394-7bfd160b-a95d-4f78-8c95-2244e66f64c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTExNzI4ODYsIm5iZiI6MTcxMTE3MjU4NiwicGF0aCI6Ii8xMDEwMDA2MjEvMzE2MDM3Mzk0LTdiZmQxNjBiLWE5NWQtNGY3OC04Yzk1LTIyNDRlNjZmNjRjMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyM1QwNTQzMDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kM2Q1YjhlMTIwNjZlNDRlYjBhNjRmNzgzMTc3ZjUxNzAzMTJlN2ZmM2EzNWJjZWQ1MmQ0MTRhZjM1ZDIwNjE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.NhxkDEvGLExMNovu9BnZNbFrfjVTEjGG0Jt2cAHw8v0)

i have not meet this issue , and how to use the augumented data?

**Comment by Aocide119** [comment]
Like muti-scale training and some test tricks.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "open-mmlab/mmrotate"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月23日(星期六) 中午1:45
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [open-mmlab/mmrotate] [Bug] DOTA Evaluation Results Feedback of Task1 is obviously lower than the official result (Issue #1013)





  
Maybe you didn't use the augumented data. By the way we encountered an issue with the DOTA V1.0 evaluation server, we can't link to the below links. 
  
i have not meet this issue , and how to use the augumented data?

**Comment by xxxyyynnn** [comment]
@jimuIee Have you solved this problem? I have the same problem as you, could you please help me?

**Comment by amomo123** [comment]
@jimuIee 你好，请问你的复现出来了吗，我也是使用mmrotate-0.3.4，跑oriented-RCNN和Redet，数据集的train和val用来训练，然后上DOTA官网测试结果和你差不多。没有改动原代码，不知道是哪里的问题。
![orcnn](https://github.com/user-attachments/assets/06e44647-2c62-42e5-89a9-4c7937065d00)

![redet](https://github.com/user-attachments/assets/ba4d2724-5392-497b-9471-4967e4c6b51e)
